guides/common/modules/proc_migrating-to-a-new-server.adoc

```diff
@@ -0,0 +1,46 @@
+[id="migrating-to-a-new-server_{context}"]
+
+.Prerequisites
+* Install a minimal {RHEL} 8 instance on the target machine with the capacity to store backup files.
+* Do not install any {RHEL} 8 software groups or third-party applications.
+
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[Performing a standard {RHEL} installation].
+
+== Creating a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 7
+
+Before you perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 system, back up your {ProjectServer} or {SmartProxyServer} data on the {RHEL} 7 system by creating an offline backup.
+
+If you recently created an offline backup, you can perform an incremental backup to update the existing backup.
+
+.Procedure
+* Perform an offline or incremental backup on the source machine:
+** To perform an offline backup, see xref:"Backing-Up-Server-and-Proxy_{context}"[].
+** To perform an incremental backup, see xref:"Performing-an-Incremental-Backup_{context}"[].
```

Comment:
Are the quotation marks here necessary? I've never seen them used in xrefs.

Reply:
Fixed